### PR TITLE
U/onsing/add scripts for physical badging connector

### PR DESCRIPTION
**Pull request by onkarsinghms**

### CODE REVIEW

File: PhysicalBadging.postman_collection.json [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+							"value": "",
+							"type": "string"
+						}
+					]
+				},
+				"method": "POST",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": "[\n\t{\n\t\t\"AssetId\":\"Mid-Sec-7\",\n\t\t\"AssetName\":\"Main Building 1st Floor Mid Section\",\n\t\t\"AccessStatus\":\"Failed\",\n\t\t\"EventTime\":\"2019-07-04T01:57:49\",\n\t\t\"UserId\":\"tifc@a830edad9050849EQTPWBJZXODQ.onmicrosoft.com\",\n\t\t\"SourceSystem\":\"Lenel\"\n\t},\n\t{\n\t\t\"AssetId\":\"Mid-Sec-7\",\n\t\t\"AssetName\":\"Main Building 1st Floor Mid Section\",\n\t\t\"AccessStatus\":\"Success\",\n\t\t\"EventTime\":\"2019-07-04T02:57:49\",\n\t\t\"UserId\":\"tifc@a830edad9050849EQTPWBJZXODQ.onmicrosoft.com\",\n\t\t\"SourceSystem\":\"Lenel\"\n\t}\n]",
```
Comment (vakataka) [inline PhysicalBadging.postman_collection.json:25]: make these postman collection params.

File: PhysicalBadging.postman_collection.json [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+		}
+	],
+	"event": [
+		{
+			"listen": "prerequest",
+			"script": {
+				"id": "6f3e901a-37a7-4fab-8188-65f7b8afa663",
+				"type": "text/javascript",
+				"exec": [
+					""
```
Comment (vakataka) [inline PhysicalBadging.postman_collection.json:60]: if this is empty, does the postman still export it ?

File: PhysicalBadging.postman_collection.json [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+		{
+			"id": "dd8b521e-8ce4-4fcd-a066-346af82c1299",
+			"key": "host",
+			"value": "https://webhook.ingestion.office.com",
+			"type": "string"
+		},
+		{
+			"id": "9eee6a64-b4dc-496d-87d0-8048e6823b06",
+			"key": "jobId",
+			"value": "f683eea3-a14b-4927-8575-44206ce70e7d",
```
Comment (vakataka) [inline PhysicalBadging.postman_collection.json:85]: use default / initial something like FILL_ME

File: README.md [hunk 4]
```diff
@@ -12,3 +11,85 @@ provided by the bot. You will only need to do this once across all repos using o
 This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
 For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
 contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+
+# m365-physical-badging-connector-sample-scripts
+This repository includes sample scripts for pushing organization's physical badging records, to be consumed by Microsoft's Insider Risk Management compliance solution.
```
Comment (vakataka) [inline README.md:16]: Ensure this text is reviewed.

File: README.md [hunk 5]
```diff
@@ -12,3 +11,85 @@ provided by the bot. You will only need to do this once across all repos using o
...
+Create you AAD APP https://docs.microsoft.com/en-us/azure/kusto/management/access-control/how-to-provision-aad-app
+
+ Note the following
+ - APP_ID (aka Application ID or Client)
+ - APP_SECRET (aka client secret)
+ - TENANT_ID (aka directory ID)
+
+## Create your job on M365 Compliance portal
+
+Provide required details at Physical Badging Connector and create a job and note the **JOB_ID**
```
Comment (vakataka) [inline README.md:31]: link to the connector documentation.

File: physicalBadgingSignals.json [hunk 6]
```diff
@@ -0,0 +1,18 @@
+﻿[
```
Comment (vakataka) [inline physicalBadgingSignals.json:1]: suggest naming the file to indicate it is a "sample data set"

File: PhysicalBadging.postman_collection.json (hunk 2 above)
Comment (onkarsinghms) [reply PhysicalBadging.postman_collection.json:60]: yes

File: README.md (hunk 4 above)
Comment (onkarsinghms) [reply README.md:16]: Gaurav reviewed the PR

**Pull request merged**